api/views.py

```diff
@@ -79,3 +79,6 @@ class UserFeedbackViewSet(ModelViewSet):
 
     def perform_create(self, serializer):
         serializer.save(reported_by=self.request.user)
+
+    def get_queryset(self):
+        return UserFeedback.objects.all()
```

Comment:
since we're not modifying the queryset we can  have it after `    serializer_class = UserFeedbackSerializer `

Reply:
Done.